Patch origin: in-house
Patch status: Solaris-specific; not suitable for upstream

This seems to be the only way to propagate these macros also
into apr-1-config.

--- apr-1.7.4/build/apr_hints.m4
+++ apr-1.7.4/build/apr_hints.m4
@@ -237,6 +237,7 @@ ;;
     *-solaris2*)
     	PLATOSVERS=`echo $host | sed 's/^.*solaris2.//'`
 	APR_ADDTO(CPPFLAGS, [-DSOLARIS2=$PLATOSVERS -D_POSIX_PTHREAD_SEMANTICS -D_REENTRANT])
+	APR_ADDTO(CPPFLAGS, [-D_LARGEFILE_SOURCE -D_FILE_OFFSET_BITS=64])
         if test $PLATOSVERS -eq 10; then
             # pthread_mutex_timedlock is broken on Solaris 10.
             # It can block without timeout in case of EDEADLK.
